src/services/api.ts

```diff
@@ -39,22 +37,22 @@ export async function delDataSource({ id, query }: { id: number, query?: string
 	return delJSON(`/datasources/${id}${query || ''}`);
 }
 
-export async function getTemplate(engine = 'dummy'): Promise<TemplateDTO> {
-	return fetchJSON(
-		`${API_HOST}/calculations/template?engine=${engine}`,
-		{ credentials: 'omit' }
-	);
-}
-
 export async function getCalculations(query?: string): Promise<void> {
 	return getJSON(`/calculations${query || ''}`);
 }
 
-export async function getCalculationsEngines(): Promise<void> {
+export async function getCalculationEngines(): Promise<string[]> {
 	return getJSON(`/calculations/engines`);
 }
 
-export async function setCalculation({ dataId, engine = 'dummy', input, workflow = 'workflow' }:
+export async function getCalculationEngine(engine = 'dummy'): Promise<EngineDTO> {
+	return fetchJSON(
+		`${API_HOST}/calculations/template?engine=${engine}`,
```

Comment:
@Valexr why do we need `API_HOST` here? It can work via BFF as well.

Reply:
ok, will try it...

Reply:
Removes version from url path to bff